libsplinter/src/peer/interconnect.rs

```diff
@@ -357,6 +366,7 @@ where
             dispatched_sender,
             recv_join_handle,
             send_join_handle,
+
```

Comment:
Spurious whitespace

Reply:
Fixed, and split out the pending loop into its own file.